.eslintrc.js

```diff
@@ -12,9 +12,37 @@ module.exports = {
     'fp/no-unused-expression': 0,
     'fp/no-nil': 0,
     'fp/no-throw': 0,
+
+    'import/extensions': [2, 'always', { ignorePackages: true }],
+    'import/newline-after-import': 2,
+    'import/no-amd': 2,
+    'import/no-anonymous-default-export': 2,
+    'import/no-cycle': [2, { commonjs: true }],
+    'import/no-deprecated': 2,
+    'import/no-dynamic-require': 2,
+    'import/no-extraneous-dependencies': 2,
+    'import/no-mutable-exports': 2,
+    'import/no-named-default': 2,
+    'import/no-namespace': 2,
+    'import/no-self-import': 2,
+    'import/no-unassigned-import': [2, { allow: ['*polyfill*', '**/*polyfill*', 'log-process-errors/**'] }],
+    'import/no-unresolved': [2, { commonjs: true }],
+    'import/no-useless-path-segments': [2, { commonjs: true }],
   },
   overrides: [
     ...overrides,
+    {
+      files: ['**/*.md'],
+      rules: {
+        'import/no-unresolved': 0,
+      },
+    },
+    {
+      files: ['**/fixtures/**/*.js'],
+      rules: {
+        'import/no-unresolved': 0,
+      },
+    },
```

Comment:
The fixtures override is specific to this project. The Markdown one should be moved to `@netlify/eslint-plugin-node`.